python_modules/dagster/setup.py

```diff
@@ -113,6 +113,7 @@ def get_version() -> str:
         "rich",
         "filelock",
         f"dagster-pipes{pin}",
+        "antlr4-python3-runtime",
```

Comment:
X

Reply:
?

Reply:
was looking for changes in dependencies/requirements trying to debug something and used this comment to link directly to this change